gradslam/structures/pointclouds.py

```diff
@@ -1340,17 +1340,28 @@ def plotly(
                 torch_colors = torch_colors * 255
             torch_colors = torch.clamp(torch_colors, min=0.0, max=255.0)
             numpy_colors = torch_colors.detach().cpu().numpy().astype("uint8")
-
-        scatter3d = go.Scatter3d(
-            x=numpy_points[..., 0],
-            y=numpy_points[..., 1],
-            z=numpy_points[..., 2],
-            mode="markers",
-            marker=dict(
```

Comment:
@NikV-JS I'd suggest refactoring it this way: Initializing a dictionary `marker = {"size": point_size}` on line 1328 before the if statement, and adding `marker["color"] =  numpy_colors` on line 1343 within `if self.has_colors and include_colors:`. And here just changing line 1349 to `marker=marker`.

Reply:
@saryazdi The suggested changes have been made. marker_dict is initialized, and depending on the include_colors condition, numpy_colors are added to the dictionary.

Reply:
@NikV-JS I'd suggest merging line 1334 and 1335 into a single line, i.e. `marker_dict = {"size": point_size}`

Reply:
They have been merged and linted.